docs/technology_stack.adoc

```diff
@@ -7,35 +7,42 @@
 toc::[]
 
 == Overview
-The following sections describe the technology stack used by Planner.
+The Planner is an Angular app built on top of Node.js. It can run as a standalone app, or be used as a library to be integrated into the link:./fabric8-platform[Fabric8 Platform].
 
-== Angular 4
-<TBD>
+Development or deployment of Planner is fairly platform agnostic, so we can skip the operating system requirements and discuss from the runtime environment to way-up.
 
-== CSS and SASS
-<WIP, to be changed to PF CSS, Less>
+== Development Runtime
+Planner uses *`Node.js`* as the platform-runtime, on top of which it builds; on that line, we use *`npm`* for resolving and distributing packages for the project.
 
-Fabric8-planner uses SASS for it's stylesheets. It also uses the Angular emulation
-of the shadow dom, hence it is recommended to place your styles in the
-*_.component.scss_* file next to the HTML and the TypeScript.
+We keep ourselves compatible with the latest stable releases of *`Node.js`* & *`npm`* - however, *`Node.js >= 6.xx`* & *`npm >= 3.xx`* releases should work and are recommended.
 
-If you want to create a shared style that could be used by multiple components, then we recommend adding it as a mixin to
-*_src/assets/stylesheets/_planner-mixins.scss_*. The mixins are imported into every
-*_.component.scss_* file.
+== Development Pipeline
```

Comment:
In the introduction of this section, I suggest we have a one liner explaining when this section comes into the development picture and a line about the subsections in this sections such as task runner, build system etc.